BubbleSort/BubbleSort.lua

```diff
@@ -0,0 +1,28 @@
+-- BUBBLE SORT --
+
+function bubbleSort(arr, sizeArr)
```

Comment:
remove sizeArr parameter you can use table.getn function, please see https://www.lua.org/pil/19.1.html